### PR TITLE
1023005

### PR DESCRIPTION
**Pull request by Johnnyeah**

### CODE REVIEW

File: whoami.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
 
 int main() {
+	printf("1023005");
```
Comment (jrjang2) [inline whoami.c]: Leave an empty line before ```return 0;```.

**Comment by jrjang2** [comment]
Great. But there is a conflict now. Please solve this issue before doing pull request.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by Johnnyeah** [comment]
I fork this project (jrjang2/lab), then git pull, it say "already up-to-date", not a "conflict ".
I delete the lab on my computer before and git clone again.
Does it result in the consequence?
So do I need to delete the github's repository and do it again?
Thank you.
p.s.The last two commits just for test.

**Comment by jrjang2** [comment]
The correct step is:

1. ```fork``` on github.
2. ```git pull --rebase``` on your work directory.

File: whoami.c [hunk 2]
```diff
@@ -1,5 +1,10 @@
 #include <stdio.h>
 
 int main() {
+	printf("1022997\n");
+	printf("1023008\n");
+	printf("1024869\n");
```
Comment (jrjang2) [inline whoami.c]: Do not modify the original code.

**Comment by jrjang2** [comment]
Merge all commits to one before doing pull request.

**Comment by jrjang2** [comment]
There is a conflict now. Please solve this issue before doing pull request. Update this pull request instead of closing this. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by Johnnyeah** [comment]
done.

**Comment by jrjang2** [comment]
Good. But there is a conflict now. Please solve this issue before doing pull request. Update this pull request instead of closing this. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by Johnnyeah** [comment]
OK

**Comment by jrjang2** [comment]
There is a conflict now. Please solve this issue before doing pull request. Update this pull request instead of closing this. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by Johnnyeah** [comment]
OK

**Comment by jrjang2** [comment]
Good!

**Pull request merged**